src/megaclient.cpp

```diff
@@ -1171,7 +1171,7 @@ MegaClient::MegaClient(MegaApp* a, Waiter* w, HttpIO* h, FileSystemAccess* f, Db
 {
     sctable = NULL;
     pendingsccommit = false;
-    tctable = NULL;
+    statusTable = NULL;
```

Comment:
```suggestion
    tctable = NULL;
    statusTable = nullptr;
```

Reply:
Definitely, it's a bug. `megacli` crashes upon login, since `tctable` points to a random memory address and it's tested upon initialization.
Fixed in my last commit